src/api/emails/index.ts

```diff
@@ -76,9 +76,35 @@ const securityCodeEmail = async (emailAddress: string, firstName: string, securi
   }
 };
 
+/**
+ * applicationSubmittedEmail
```

Comment:
Can we write one line description of the function please instead of function name.